README.md

```diff
@@ -18,15 +18,18 @@ Targets:
   ftw                runs ftw tests with a built plugin and Envoy.
   lint               verifies code quality.
   runExample         spins up the test environment, access at http://localhost:8080.
+  runExampleD        spins up the test environment in detached mode, access at http://localhost:8080.
```

Comment:
I don't know if detached is so helpful, it's always  possible to just run the command in another terminal I think.

Problems can come up because of a leftover detached docker process so unless required I wouldn't add an official command for it.

Reply:
Marking as resolved (via https://github.com/corazawaf/coraza-proxy-wasm/pull/238)